website/blog/2023-04-26-deprecating-dbt-metrics.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Why we're deprecating dbt_metrics"
```

Comment:
```suggestion
title: "Why we're deprecating the dbt_metrics package"
```

Just a suggestion in case folks don't have a familiarity with dbt_metrics, that they know metrics themselves aren't going anywhere, just the package

Reply:
Or even something like `Why we're deprecating the dbt_metrics: The future of metrics in dbt is with MetricFlow`